openedx/core/djangoapps/theming/core.py

```diff
@@ -21,9 +22,14 @@ def enable_theming():
         )
 
     for theme in get_themes():
-        locale_dir = theme.path / "conf" / "locale"
-        if locale_dir.isdir():
-            settings.LOCALE_PATHS = (locale_dir, ) + settings.LOCALE_PATHS
-
```

Comment:
Although, we should append instead of prepending the theme specific LOCALE_PATHS.